astropy/coordinates/tests/test_solar_system.py

```diff
@@ -429,8 +429,8 @@ def test_url_or_file_ephemeris(time):
 def test_url_ephemeris_wrong_input():
     # Try loading a non-existing URL:
     time = Time('1960-01-12 00:00')
-    with pytest.raises(HTTPError):
-        get_body('earth', time, ephemeris='http://data.astropy.org/path/to/nonexisting/file.bsp')
+    with pytest.raises((HTTPError, URLError)):
+        get_body('earth', time, ephemeris=get_pkg_data_filename('path/to/nonexisting/file.bsp'))
```

Comment:
This one looks wrong.. This is a test for giving an invalid URL to the `get_body`

Reply:
Yes, but if the server for that URL does not respond there is no way to test for the expected HTTPError any way, so the only alternative would be to continue let it fail the entire run (or perhaps allow it to xfail).

Reply:
I'm OK with reverting the change (maybe changing the URL to http://www.astropy.org/astropy-data... which might be more stable), but then all the expected exceptions should perhaps be tested.
Notably
https://github.com/astropy/astropy/blob/1bf53dcf5672d5808580339725e301a587185d73/astropy/coordinates/solar_system.py#L173-L176
is probably never ever raised, since `urlparse` accepts almost any string without raising; passing a broken path in fact fails in
```python
  File "/opt/lib/python3.10/site-packages/astropy/utils/data.py", line 1729, in _url_to_dirname
    raise ValueError(f"Malformed URL: '{url}'")
ValueError: Malformed URL: 'www.astropy.org/astropy-data/path/to/nonexisting/file.bsp'
```
so perhaps replace the `urlparse` attempt with `utils.data.is_url`?